packages/admin/admin-stories/src/admin/alert/Alert.tsx

```diff
@@ -6,6 +6,10 @@ import { Stack } from "@mui/system";
 import { storiesOf } from "@storybook/react";
 import React from "react";
 
+function dummyOnClose(): string {
+    return "";
+}
```

Comment:
This shouldn't return a string 😅 And for better understandability, please define it inline:

```tsx
```tsx
<Alert
    severity="info"
    title="Title"
    button={
    action={
        <Button variant="text" startIcon={<ArrowRight />}>
            Action Text
        </Button>
    }
    onClose={() => {
        // noop
    }
>
    Notification Text
</Alert>
```

```